scipy/stats/tests/test_bootstrap.py

```diff
@@ -470,3 +470,33 @@ def statistic_1d(*data):
     res1 = statistic(x, y, z, axis=axis)
     res2 = statistic2(x, y, z, axis=axis)
     assert_allclose(res1, res2)
+
+
+@pytest.mark.parametrize("method", ["basic", "percentile", "BCa"])
```

Comment:
I will mark this xslow because it's not really important to run this regularly. I'm leaving it unmarked for now just to see that it passes.